src/components/Navbar/WalletButton.tsx

```diff
@@ -102,38 +121,80 @@ export const WallettButtons = () => {
           </Typography>
         </ConnectButton>
       ) : (
-        <WalletMenuButton
-          id="wallet-digest-button"
-          onClick={handleWalletMenuClick}
-        >
-          {isSuccess && activeChain ? (
-            <WalletMgmtBadge
-              overlap="circular"
-              className="badge"
-              anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
-              badgeContent={
-                <WalletMgmtChainAvatar
-                  src={activeChain?.logoURI || ''}
-                  alt={'wallet-avatar'}
-                >
-                  {activeChain.name[0]}
-                </WalletMgmtChainAvatar>
-              }
-            >
-              <WalletMgmtWalletAvatar
-                src={getConnectorIcon(account.connector)}
+        <Stack direction="row" spacing={2}>
+          <WalletMenuButton id="wallet-digest-button" onClick={handleXPClick}>
+            <Image
+              src={`https://effigy.im/a/${account?.address ?? 'jumper.eth'}.png`}
+              alt="Effigy Wallet Icon"
+              width={28}
+              height={28}
+              priority={false}
+              unoptimized={true}
+              style={{
+                borderRadius: '100%',
+                borderStyle: 'solid',
+                borderWidth: '2px',
+                borderColor:
+                  theme.palette.mode === 'light'
+                    ? theme.palette.white.main
+                    : alpha(theme.palette.white.main, 0.08),
+              }}
+            />
+            {points === undefined ? (
+              <Skeleton
+                variant="text"
+                sx={{
+                  fontSize: { xs: 24, sm: 24 },
+                  minWidth: 25,
+                  marginRight: 1.1,
+                  marginLeft: 1.1,
+                }}
               />
-            </WalletMgmtBadge>
-          ) : null}
-          <Typography
-            variant={'lifiBodyMediumStrong'}
-            width={'auto'}
-            marginRight={0.25}
-            marginLeft={0.75}
+            ) : (
+              <Typography
+                variant={'lifiBodyMediumStrong'}
+                width={'auto'}
+                marginRight={1.1}
+                marginLeft={1}
+              >
+                {points}
+              </Typography>
+            )}
+            <XPIcon size={28} />
+          </WalletMenuButton>
```

Comment:
Maybe this whole section could become it´s own styled-component?

Reply:
we can do it, but then i would create it's own styled-component for connected button as well; @tcheee what is your take here